.github/scripts/patch.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -x
+
+exit 1 # The following values must be change before every new patch release
+OTEL_JAVA_VERSION=1.21.0
```

Comment:
This has to be updated before every new release. I opted to create this script instead of adding github steps because I have to checkout a specific tag and create a new branch and apply the patch. These set of operations would not go well with the github actions.